src/ceph-volume/ceph_volume/api/lvm.py

```diff
@@ -1196,23 +1233,37 @@ def remove_lv(lv):
     return True
 
 
-def is_lv(dev, lvs=None):
+def is_lv(dev=None, lv_name=None, lvs=None):
     """
     Boolean to detect if a device is an LV or not.
     """
-    splitname = dmsetup_splitname(dev)
-    # Allowing to optionally pass `lvs` can help reduce repetitive checks for
-    # multiple devices at once.
-    if lvs is None or len(lvs) == 0:
-        lvs = Volumes()
+    if dev and lv_name:
+        raise RuntimeError('is_lv: can\'t pass both dev and lv_name')
```

Comment:
These errors bubble up on the command line. Having this error shown to a user would be confusing.  Does it really need to have this sort of error handling?

Reply:
I was intending to use these changes at [[1]](https://github.com/ceph/ceph/pull/30102/files#diff-0d937eeea907d69c580b4c2fe1fdf096R499) and [[2]](https://github.com/ceph/ceph/pull/30102/files#diff-0d937eeea907d69c580b4c2fe1fdf096R859). I'll get that done...

Reply:
Please review the change. I've replaced the current message with "internal error message" - https://github.com/ceph/ceph/pull/30102/files#diff-0d937eeea907d69c580b4c2fe1fdf096R1238.